lib/python/ensembl/brc4/runnable/amend_genome_metadata.py

```diff
@@ -0,0 +1,187 @@
+#!env python3
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+from pathlib import Path
+import re
+from typing import Any, Dict, List, Tuple
+import eHive
+import gzip
+import csv
+
+from Bio import SeqIO
+from ensembl.brc4.runnable.utils import Utils
+
+
+class MissingDataError(Exception):
+    """Used if some data is missing from the report file."""
+    def __init__(self, report_path: str, accession: str, msg):
+        report_msg = f"Can't get data for {accession} in report {report_path}"
+        if msg:
+            report_msg = f"{report_msg}: {msg}"
+        self.msg = report_msg
+
+
+class amend_genome_metadata(eHive.BaseRunnable):
+    """Runnable to add things to the genome metadata, if any.
+
+    Params:
+        genome_data: Genome data following the schema/genome_schema.json.
+        work_dir: Directory where the new file will be created.
+        report: Path to the INSDC/RefSeq sequences report to parse.
+        gbff: Path to the INSDC/RefSeq gbff file to parse.
+        brc4_mode: Activate BRC4 mode (default).
+    
+    Dataflows:
+        2: One dict with 2 keys:
+            metadata_type: 'genome'.
+            metadata_json: Path to the new genome json.
+    
+    Notes:
+        BRC4 mode does the following:
+            NOTHING FOR NOW
+    """
+
+    def param_defaults(self):
+        return {
+            "brc4_mode": True,
+        }
+
+    def run(self):
+        genome_data = self.param('genome_data')
+        work_dir = self.param('work_dir')
+        report_path = self.param('report')
+        gbff_path = self.param('gbff')
+        brc4_mode = self.param('brc4_mode')
+
+        # Create dedicated work dir
+        if not os.path.isdir(work_dir):
+            os.makedirs(work_dir)
```

Comment:
```suggestion
        if not work_dir.is_dir():
            work_dir.mkdir(parents=True)
```
The `parents` flag is set to `True` to make sure the behaviour is the same as `os.makedirs`.